lib/src/Uncertainty/Distribution/FisherSnedecorFactory.cxx

```diff
@@ -76,7 +94,7 @@ FisherSnedecor FisherSnedecorFactory::buildAsFisherSnedecor(const Sample & sampl
 
   // override starting point
   OptimizationAlgorithm solver(factory.getOptimizationAlgorithm());
-  solver.setStartingPoint(parametersLowerBound);
+  solver.setStartingPoint(parametersFromMoments);
```

Comment:
parametersLowerBound is unused

Reply:
No: parametersLowerBound is used later, to set the lower bounds of the optimization problem in the MaximumLikelihoodFactory:

https://github.com/openturns/openturns/blob/60dbd347039aba890dd3b68dcf833f02780ebdbf/lib/src/Uncertainty/Distribution/FisherSnedecorFactory.cxx#L101